resume-buddy/src/test/java/com/google/sps/MatchTest.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.sps.data.Pair;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests for matching algorithm */
+@RunWith(JUnit4.class)
+public class MatchTest {
```

Comment:
This test file should be in

`resume-buddy/test/java/com/google/sps/MatchTest.java`

instead of

`resume-buddy/src/test/java/com/google/sps/MatchTest.java`

because `src/` should only contain source code.

Reply:
Tests don't run when `/test` is not under `/src`. 
I think in Maven project directory layout test is supposed to be inside src. http://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html
We also had it similar way for starter projects.

Reply:
Thanks for sharing! Makes sense